libraries/Microsoft.Bot.Builder.Dialogs.Adaptive/Actions/HttpRequest.cs

```diff
@@ -332,6 +339,13 @@ public override async Task<DialogTurnResult> BeginDialogAsync(DialogContext dc,
                     requestResult.Content = content;
                     break;
 
+                case ResponseTypes.BinaryData:
+                    // Try to resolve binary data
+                    var bytes = await response.Content.ReadAsByteArrayAsync();
+                    var encodedString = Convert.ToBase64String(bytes);
```

Comment:
>ToBase64String [](start = 48, length = 14)

Why not return as byte[] ?

Reply:
> > ToBase64String [](start = 48, length = 14)
> 
> Why not return as byte[] ?

return byte[] now, user could use LG function Base64 to parse the data.